seqr/views/utils/permissions_utils.py

```diff
@@ -76,22 +76,26 @@ def _map_anvil_seqr_permission(anvil_permission):
 def anvil_has_perm(user, permission_level, project):
     if not project_has_anvil(project):
         return False
-    return has_workspace_perm(user, permission_level, project.workspace_namespace, project.workspace_name)
+    return bool(has_workspace_perm(user, permission_level, project.workspace_namespace, project.workspace_name))
```

Comment:
The signature for `has_workspace_perm`should not change, if you don't pass `meta_fields` it needs to continue to return a bool

Reply:
I see what you mean.